std/unicode.zig

```diff
@@ -155,6 +156,18 @@ pub const Utf8View = struct {
         return initUnchecked(s);
     }
 
+    pub fn eql(self: &const Utf8View, other: &const Utf8View) bool {
```

Comment:
I think `mem.eql` should be used when you don't care about encoding and only about memory, while something like `unicode.eql` should take into account that a "letter" can be a combination of many unicode code points, and that the order of these code points, don't always matter.

Reply:
I did not realise that, I'll look into it and build a proper solution :0